encoding/mvt/layer_internal_test.go

```diff
@@ -0,0 +1,137 @@
+package mvt
+
+import (
+	"context"
+	"testing"
+
+	"github.com/go-spatial/geom"
+	"github.com/arolek/p"
+	vectorTile "github.com/go-spatial/geom/encoding/mvt/vector_tile"
```

Comment:
Add the `github.com/arolek/p` as a dependency to the list of dependencies in [README.md](https://github.com/go-spatial/geom/blob/master/README.md)

Reply:
by using the static variables, this file no longer imports `p` but `layer_test.go` does, I'll add it